test/service/test_manifest.py

```diff
@@ -82,6 +86,24 @@ def _get_manifest_url(self, format_, filters):
         service = ManifestService(StorageService())
         return service.get_manifest(format_, filters)
 
+
+class TestManifestEndpoints(ManifestTestCase):
+
+    def run(self, result: Optional[unittest.result.TestResult] = ...) -> Optional[unittest.result.TestResult]:
+        """
+        This patch introduces a context manager at the unittest level `run`. Allowing for
+        test in this suite to 'run' in the patched context.
+        The context manager patches 'ManifestService._can_use_cached_manifest' in order to
+        prevent a cached manifest from being used. It allows for testing of manifest
+        generation while ignoring cache functionality.
+        """
+        with mock.patch(
+            'azul.service.manifest_service.ManifestService._can_use_cached_manifest'
+        ) as mock_can_use_cached_manifest:
+            mock_can_use_cached_manifest.return_value = False
+            run_ = super().run(result)
```

Comment:
```suggestion
            return super().run(result)
```

Be sure to understand why the variable is unnecessary.